src/main/daml/Daml/Finance/Interface/Account/Account.daml

```diff
@@ -53,7 +54,7 @@ toKey : View -> AccountKey
 toKey v = AccountKey with custodian = v.custodian; owner = v.owner; id = v.id
 
 -- | An interface which represents an established relationship between a provider and an owner.
-interface Account requires Disclosure.I where
+interface Account requires Lockable.I, Disclosure.I where
```

Comment:
I presume we have ordering here like (ie., like we do for `import`) :

```suggestion
interface Account requires Disclosure.I, Lockable.I where
```

Reply:
We rather order according to `requires`, in particular, `Lockable.I` requires `Disclosure.I`. As an example, we have:
`interface Fungible requires Transferable.I, Base.I, Lockable.I, Disclosure.I where`